backend/server/src/models/tournamentModel.ts

```diff
@@ -15,6 +25,13 @@ export interface Tournament {
   tournamentType: TournamentType;
   organizerEmail?: string;
   organizerPhone?: string;
+  maxPlayers: number;
+  players: string[]; // Array of player identifiers (userID from user objects)
+  matchSchedule: Match[];
```

Comment:
Should be a Reference to the `Match` ? When we query for the tournament, we probably want to return at least the ID's of the matches in the tournament.

Reply:
We could  then use [population](https://mongoosejs.com/docs/populate.html) when querying the Tournaments